extensions/amp-consent/0.1/consent-ui.js

```diff
@@ -651,6 +683,7 @@ export class ConsentUI {
     setStyles(this.parent_, {
       transform: '',
       transition: '',
+      '--lightbox-height': '',
```

Comment:
Why do we need to reset a custom prop? I would imagine it's unused?

Reply:
That's fair.